acceptance/run.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd $(dirname $0)/..
+build/builder.sh make install
```

Comment:
we should wipe the certs/ dir here. cockroach create-ca and create-node fails if the files exist.